data/os/Debian.yaml

```diff
@@ -0,0 +1,3 @@
+---
+otelcol::config_file_owner: otel
+otelcol::config_file_group: otel
```

Comment:
A configuration file owner by the user that run the service looks like a smell.  Is this really required?

"Usual" `user:group:mode` are `root:root:644` or if sensitive info is in the file `root:service_group:640`.  If the service own the file it can change its permissions and rewrite it.  Bad designed software does this.  Vulnerable software being pwned too :stuck_out_tongue_closed_eyes: